src/odemis/acq/test/acq_test.py

```diff
@@ -486,5 +484,306 @@ def on_progress_update(self, future, start, end):
         self.end = end
         self.updates += 1
 
+class CRYOSECOMTestCase(unittest.TestCase):
+    backend_was_running = False
+
+    @classmethod
+    def setUpClass(cls):
+
+        try:
+            test.start_backend(ENZEL_CONFIG)
+        except LookupError:
+            logging.info("A running backend is already found, skipping tests")
+            cls.backend_was_running = True
+            return
+        except IOError as exp:
+            logging.error(str(exp))
+            raise
+
+        # create some streams connected to the backend
+        cls.ccd = model.getComponent(role="ccd")
+        cls.light = model.getComponent(role="light")
+        cls.light_filter = model.getComponent(role="filter")
+        cls.ebeam = model.getComponent(role="e-beam")
+        cls.sed = model.getComponent(role="se-detector")
+        cls.SEM_focuser = model.getComponent(role="ebeam-focus")
+        cls.FM_focuser = model.getComponent(role="focus")
+
+        cls.FM_focus_pos = 0.5e-6  # arbitrary current focus position
+        cls.SEM_focus_pos = 0.1  # arbitrary current position
+
+    @classmethod
+    def tearDownClass(cls):
+        if cls.backend_was_running:
+            return
+        del cls.light
+        test.stop_backend()
+
+    def setUp(self):
+        if self.backend_was_running:
+            self.skipTest("Running backend found")
+
+        self._nb_updates = 0
+        self.streams = []
+
+        self.FM_focuser.moveAbs({"z": self.FM_focus_pos}).result()
+        self.SEM_focuser.moveAbs({"z": self.SEM_focus_pos}).result()
+
+    @unittest.skip("Only zstack")
+    def test_only_FM_streams_without_zstack(self):
+        # create streams
+        s1 = stream.FluoStream(
+            "fluo1", self.ccd, self.ccd.data, self.light, self.light_filter
+        )
+        s1._focuser = self.FM_focuser
```

Comment:
You should set the focuser by passing `focuser = self.fm_focuser` in the initialisation.